e2e/pages/administration/permissions.po/index.ts

```diff
@@ -0,0 +1,83 @@
+import { browser } from 'protractor';
+import { elements, baseUrl, names, columns } from './constants';
+import { User } from '../../../../src/app/shared/models/user';
+import { AdministrationBase } from '../base.po';
+import { LocalPermissions } from '../../../../src/app/shared/models/LocalPermissions';
+
+export class PermissionsAdministration extends AdministrationBase {
+  constructor() {
+    super(elements.uniqueElement, names.pageName);
+  }
+
+  public columns = columns;
+
+  navigateTo() {
```

Comment:
I am not sure you need this method, because it already exists in base

Reply:
It does not